src/main/java/github/scarsz/discordsrv/api/commands/SlashCommandPriority.java

```diff
@@ -0,0 +1,66 @@
+/*-
+ * LICENSE
+ * DiscordSRV
+ * -------------
+ * Copyright (C) 2016 - 2021 Austin "Scarsz" Shapiro
+ * -------------
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * END
+ */
+
+package github.scarsz.discordsrv.api.commands;
+
+import lombok.Getter;
+
+/**
+ * <p>Used to indicate the registration priority of slash commands and the invocation order of slash command handlers</p>
+ * <p>Defaults to {@link #NORMAL} in {@link PluginSlashCommand} where it's not specifically set</p>
+ * <p>Defaults to {@link #NORMAL} in {@link SlashCommand} annotations where it's not specifically set</p>
+ */
+public enum SlashCommandPriority {
```

Comment:
Is there a reason that the existing API's EventPriority enum couldn't be used? It's enum values are more self explanatory as well

Reply:
In my opinion, using "Listener" priorities to indicate the registration order of slash commands seems a bit weird and misleading, since they aren't listeners. The existing java doc in the listener priority class would also be very lengthy if we have to add two more uses.

Reply:
Forgot that Enum's name is ListenerPriority. In my opinion that enum should be deprecated in favor of a generic enum like EventPriority. The documentation wouldn't really be an issue.

Reply:
There is also another problem, for slash command handlers, an "important" handler should run first, as opposed to bukkit events where "important" handlers run last. Using the same enum for both cases would be very confusing.

Reply:
The only way to make changes to ListenerPriority would be to break the API or make a annotation to replace `@Subscribe`. I'm in favor of moving toward a single more generic enum like "Order", though

Reply:
`@Subscribe` is a pretty big part of the API though, is now the right time to break it? I feel like DiscordSRV v2 would be better timing for that.